libs/storage/Tsavorite/cs/src/core/Allocator/MallocFixedPageSize.cs

```diff
@@ -76,21 +73,19 @@ public class MallocFixedPageSize<T> : IDisposable
         private readonly T[][] values = new T[LevelSize][];
         private readonly IntPtr[] pointers = new IntPtr[LevelSize];
 
-        private readonly int RecordSize;
-
         private volatile int writeCacheLevel;
 
         private volatile int count;
 
-        internal bool IsPinned => isPinned;
-        private readonly bool isPinned;
+        internal static int RecordSize => Unsafe.SizeOf<T>();
+        internal static bool IsPinned => Utility.IsBlittable<T>();
 
         private const bool ReturnPhysicalAddress = false;
```

Comment:
Could we remove this knob too? I checked through the FASTER git history and it seems to be unused.

Reply:
Yes, go ahead and remove it

Reply:
Yes, let's remove these. Haven't needed in all these years. Can always resurrect if necessary later.

Reply:
We could simplify the class even further by removing support for non-blittable types, which are only used by a test where T is `IHeapContainer<Value>`. Might be better to leave that as potential follow-up, as there is more proper constraining opportunities in the allocator design elsewhere.